downstairs/src/lib.rs

```diff
@@ -1797,7 +1797,7 @@ pub struct ActiveUpstairs {
  */
 #[derive(Debug)]
 pub struct Downstairs {
-    pub region: Arc<Region>,
```

Comment:
Curious, why this change too?

Reply:
Nevermind this question, I get it now - otherwise `dirty_extents` would have to be guarded with a Mutex or something.

Reply:
Yeah basically, its because i couldnt make some func take an `&mut self` and still call it where i wanted. and nothing actually was using the Arc behavior, like no code ever cloned the reference, so i just removed the arc and everything was still happy

Reply:
Not blocking this review, but I'm wondering what it would take to keep this. I have a branch that changes the downstairs to operate on requests on parallel, which will clone this reference.